ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-commands.sh

```diff
@@ -15,7 +15,10 @@ ES_PASSWORD=$(cat "/secret/password")
 ES_USERNAME=$(cat "/secret/username")
 
 # Clone the e2e repo
-git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+REPO_URL="https://github.com/cloud-bulldozer/e2e-benchmarking";
+LATEST_TAG=$(curl -s "https://api.github.com/repos/cloud-bulldozer/e2e-benchmarking/releases/latest" | jq -r '.tag_name');
```

Comment:
we had issues on jenins being able to consistently connect and get the latest tag. I wonder if we will hit that here as well. Might need another option

Reply:
+1 Also, do we even want a latest tag vs latest i.e. git clone of main branch?

Reply:
That shouldn't ideally happen here as it is just hitting the endpoint and getting the tag_name. Where as previous command had some processing like below
```
$(curl -s https://api.github.com/repos/cloud-bulldozer/kube-burner/releases/latest | jq -r '.assets | map(select(.name | test("linux-x86_64"))) | .[0].browser_download_url')
```
And it used to run `tar` on top of it to extract binaries too.

Reply:
That shouldn't ideally happen here as it is just hitting the endpoint and getting the tag_name. Where as previous command had some processing like below
```
$(curl -s https://api.github.com/repos/cloud-bulldozer/kube-burner/releases/latest | jq -r '.assets | map(select(.name | test("linux-x86_64"))) | .[0].browser_download_url')
```
And it used to run `tar` on top of it to extract binaries too.

Reply:
So if someone wants a specific tag, they can override `E2E_VERSION` in their test or else `latest` would be picked by default.